django_ecommerce/settings.py

```diff
@@ -27,7 +27,7 @@
 
 ALLOWED_HOSTS = []
 
-
+LOGIN_REDIRECT_URL = "/"
```

Comment:
It would be better if we could name it 
```suggestion
BASE_URL = "/"
```

Reply:
Done